notebooks/tuning.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>I think its quite nice that you plot the example windows. Will be useful for people to visualize what we mean with these windows/FHs.  Wondering - can/should we move some of these graphs to the intro? I talk a bit about Forecasting horizons, but really like this visualization. Happy either way, just leaving comment here so we can discuss ordering. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-pydata-global-2022/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I think the cross-validation is stronlgy related to grid search and backtesting with <code>evaluate</code> which I both show in this notebook. So actually I would prefer to have it therefore all together at this place, but sure lets discuss this maybe in the call then</p>